### DISCUSSION TITLE
Newbie here… how do I…….?

### DISCUSSION THREAD

**Discussion by ETHeOReDrheTorICaL** [djsime1/awesome-flipperzero#90] [General]
Hi….. how do I execute a “pull request” in order to get firmware…. Repositories (in full) .. meaning all of them … onto my flipper device.

**Comment by djsime1** [comment]
Actually, the terminology of a "Pull request" isn't what you think. If someone makes a copy of this repository, they can make changes and add their own items. If they want to see their own changes in my 'official' repository, a Pull Request gets opened. In context, the person is _requesting_ that I _pull_ their changes into this repository. Thus, you don't need to open a pull request if you wish to download anything!

**Comment by djsime1** [comment]
Also, chances are you aren't interested in downloading EVERYTHING in this repository to your Flipper. I have entries in the [FAQ document](https://github.com/djsime1/awesome-flipperzero/blob/main/FAQ.md) regarding [downloading dump files](https://github.com/djsime1/awesome-flipperzero/blob/main/FAQ.md#how-do-i-install-databases-and-dumps) and [installing custom firmwares](https://github.com/djsime1/awesome-flipperzero/blob/main/FAQ.md#how-do-i-install-custom-firmwares). Essentially, you'll need to click on each link in the list you're interested in downloading, then read that item's repository README file/description and see if theres any instructions on how to use the resources. In fact, a lot of the links are meant to work on your computer, not on the Flipper.
